recipes/stripy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "stripy" %}
+{% set version = "v2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/underworldcode/{{ name|lower }}/archive/refs/tags/{{ version }}.tar.gz 
+  sha256: 6e40a36527757f99c5f1bb1220109c8e42281b274fef7fac6da1a8e8d240d67b 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - numpy
+    
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >=1.0.0
+
+test:
+  requires:
+    - pytest 
+    - setuptools # for pkg_resources
+
+  source_files:
+    - tests
+
+  commands:
+    - pytest
+
+about:
+  home: "https://github.com/underworldcode/stripy"
+  license: LGPL-2.0-or-later
+  license_family: GPL
+  license_file: COPYING.LESSER
+  summary: "Python interface to TRIPACK and STRIPACK fortran code for triangulation/interpolation in Cartesian coordinates and on a sphere"
+  doc_url: https://underworldcode.github.io/stripy/2.0.5b2
+  dev_url: https://github.com/underworldcode/stripy
+
+extra:
+  recipe-maintainers:
+    - lmoresi
+    - brmather
```

Comment:
@rbeucher, since you listed them as maintainers, you need to ask them to comment here saying they are okay with being maintainers

Reply:
OK. @lmoresi, @brmather. We need you here!

Reply:
I am happy to be a maintainer.